README.md

```diff
@@ -10,6 +10,25 @@ This repository contains code to train and score a Spatial Transformer Network o
 
 >  It can be inserted into existing convolutional architectures, giving neural networks the ability to actively spatially transform feature maps, conditional on the feature map itself, without any extra training supervision or modification to the optimisation process[1].
 
+
+## Model Metadata
+| Domain | Application | Industry  | Framework | Training Data | Input Data Format |
+| ------------- | --------  | -------- | --------- | --------- | -------------- |
+| Image | Computer Vision | General | TensorFlow | [Cluttered MNIST Sample](https://github.com/daviddao/spatial-transformer-tensorflow/blob/master/data/mnist_sequence1_sample_5distortions5x5.npz) | Images |
```

Comment:
Instead of just "Images" maybe we should specify what kind of images work? E.g. file type and maximum dimensions (if any)